app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -102,6 +105,19 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         .observeOn(mainThread())
         .takeUntil(screenDestroys)
         .subscribe { uiChange -> uiChange(this) }
+
+    illustrationImageView.setImageResource(illustrationResourceId())
+  }
+
+  private fun illustrationResourceId(): Int {
+    val today = LocalDate.now()
```

Comment:
Can you provide the `userClock` here instead of using the `.now()` method? This will compute the dates in the utc clock instead, which is not what we want.

Reply:
done